angular/projects/public-nrpti/src/app/records/records-list/records-list.component.html

```diff
@@ -1,66 +1,54 @@
 <main class="container-fluid-padding">
   <div *ngIf="!loading">
+
+    <div class="row mt-0 px-4 justify-content-center">
+      <h1>
+        Natural Resources Compliance and Enforcement Database
+      </h1>
+
+      <p>
+        This site can be used to search for compliance and enforcement actions taken by natural resource agencies
+        including inspections, orders, violation tickets, administrative penalties, court convictions, and more. Use the
+        keyword search bar or filter your resultes for more specific records.
+      </p>
+
+    </div>
+
     <div class="row">
-      <div class="col-xl-3 col-lg-4 order-lg-1 order-2 mb-3">
-        <div class="search-filter__header">
+      <div class="col-lg col-xl-5 mb-3 d-flex flex-column">
+        <div class="sticky-search">
+        <div class="search-filter__header pt-3 order-2 order-lg-1">
```

Comment:
add `col` to the class attribute to make the textbox same width as the advanced filter box